uw-frame-components/js/frame-config.js

```diff
@@ -32,6 +32,7 @@ define(['angular'], function(angular) {
             'crestalt': 'UW Crest',
             'group': 'UW-Madison',
             'mascotImg': 'img/bucky.gif',
+            'profileUrl': '/profile/emergencyInfo',
```

Comment:
It's probably not going to be appropriate to show as-currently-implemented `/profile/emergencyInfo` as the profile link for all Madison users, since alas as currently implemented that registrar-and-dean-of-students-office project is scoped to only "students". Staff directed there as it presently stands will likely be confused and think that their registering emergency information there might have some useful effect when it won't.

This means we should circle back and make `/profile` degrade more gracefully across user roles, so that it would become appropriate to send all users there.